l3experimental/l3draw/l3draw.dtx

```diff
@@ -848,7 +848,7 @@
 %   This is a shortcut for \cs{draw_path_ellipse:nnn}.
 % \end{function}
 %
-% \begin{function}{\draw_path_rectangle:nn, \draw_path_rectangle_corners:nn}
+% \begin{function}[updated = 2024-03-02]{\draw_path_rectangle:nn, \draw_path_rectangle_corners:nn}
```

Comment:
There's no change in the documented API, so we don't normally change the dates.

Reply:
I confused myself the 'behavior' and 'expected behavior' again. 😢 `updated` is only needed when the expected behavior changes, is it?

Reply:
@muzimuzhi  Yes, exactly